README.md

```diff
@@ -23,33 +24,18 @@ More information:
 
 ## Contents
 
-* [What's new in Zowe Explorer 1.9.0](#what's-new-in-zowe-explorer-1.9.0)
+* [What's new in Zowe Explorer 1.10.0](#what's-new-in-zowe-explorer-1.10.0)
 * [Prerequisites](#prerequisites)
 * [Getting Started](#getting-started)
 * [Credentials Security](#credentials-security)
 * [Usage Tips](#usage-tips)
 * [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.9.0
+## What's new in Zowe Explorer 1.10.0
 
 New features:
 
-* Added the Allocate Like feature.
-* Added the ability to disable/enable profile validation.
-* Added the ability to access other profiles during profile validation.
-
-Enhancements:
-
-* Datasets and members names are displayed in all uppercase letters even if typed in lowercase letters.
-* Grouped Favorites by profile for Datasets, USS, and Jobs.
-* Other profiles can be accessed during profile validation.
-* Updated the environment check for Theia compatibility.
-
-Bug Fixes:
-
-* Fixed USS renaming issues.
-* Fixed the deletion of datasets issue.
-* Removed errors in Favorites items caused by profiles that are created by other extensions.
+* Added base profile support. You can now log in to the API Mediation Layer via Zowe CLI and use that token with a service profile in Zowe Explorer.  
```

Comment:
"...that token to authenticate a service profile with the same host & port"?

Reply:
What is "that token"?

Reply:
the token the user got in their base profile after login 😋